release/nightly_tests/multimodal_inference_benchmarks/video_object_detection/daft_main.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
```

Comment:
Note as comment - This file is adapted from https://github.com/Eventual-Inc/Daft/tree/9da265d8f1e5d5814ae871bed3cee1b0757285f5/benchmarking/ai/video_object_detection